recipes/sciunit/meta.yaml

```
@@ -0,0 +1,73 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set version = "0.2.2.4" %}
+
+package:
+  name: sciunit
+  version: {{ version }}
+
+source:
+  url: https://github.com/scidash/sciunit/archive/{{ version }}.tar.gz
+  sha256: e372371ee6686cf9110d2142e1388336d91297a9140c9ac2e3323f89187ea374
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
```

This looks like it can be `noarch: python`:
```suggestion
  noarch: python
```